app/controllers/application_controller.rb

```diff
@@ -16,4 +16,17 @@ def render_json_message(status, options = {})
   def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:sign_up, keys: %i[first_name last_name phone])
   end
+
+  def after_sign_in_path_for(resource)
+    if resource.is_a?(AssetManager)
```

Comment:
can you turn this into a switch statement? It will look cleaner :) also why does `is_a` sometimes have `?` and sometimes not?